### ISSUE TITLE
Snap to maximize failed when applied to browsers in Stage Manager

### ISSUE THREAD

**Issue by kaiyo655** [rxhanson/Rectangle#1209]
With Stage Manager enabled , Chrome and Safari, when using snap to maximize, ignore the Stage Manager's edge settings and become fully maximized in almost fullscreen.

macOS version: Sonoma
Rectangle version: 0.69
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 
2023-07-21T12:40:09+08:00: AX position proposed: (0.0, 107.0), result: (0.0, 107.0)
2023-07-21T12:40:09+08:00: AX sizing proposed: (1192.0, 944.0), result: (1192.0, 944.0)
2023-07-21T12:40:10+08:00: AX sizing proposed: (1512.0, 944.0), result: (1374.0, 944.0)
2023-07-21T12:40:10+08:00: AX position proposed: (0.0, 38.0), result: (0.0, 38.0)
2023-07-21T12:40:10+08:00: AX sizing proposed: (1512.0, 944.0), result: (1512.0, 944.0)
2023-07-21T12:40:10+08:00: maximize | display: (0.0, 0.0, 1512.0, 944.0), calculatedRect: (0.0, 38.0, 1512.0, 944.0), resultRect: (-0.0, 38.0, 1512.0, 944.0), srcScreen: 内建视网膜显示器, destScreen: 内建视网膜显示器, resultScreen: 内建视网膜显示器
2023-07-21T12:40:15+08:00: Unable to obtain the accessibility element with the specified attribute at mouse location

**Comment by spalger** [comment]
I've just started using stage manager and really love it, but I needed to export my settings, then dive into the rectangle.json file and customize the following settings:

```json
"stageSize": {
  "float": 235
},
"screenEdgeGapBottom": {
  "float": 70
},
"screenEdgeGapLeft": {
  "float": 0
},
"screenEdgeGapRight": {
  "float": 25
},
"screenEdgeGapTop": {
  "float": 130
},
```

This allows me to full screen apps and see a little bit of the widgets on the top of the screen, the dock on the right side, the recent apps on the left, and a little bit of my desktop background along the bottom (useful for clicking to clear out windows and seeing my whole desktop).

![CleanShot 2023-07-21 at 07 44 19](https://github.com/rxhanson/Rectangle/assets/1329312/dbdaa38b-a346-4a15-a43e-f8bf392a47c2)

I feel like this should probably be something that's easier to configure in the UI, but I understand that this app is already a whole bunch of knobs and customizations.

**Comment by rxhanson** [comment]
The stage manager issue has been reported elsewhere, but I do have it slated for the next release (it's something I cannot reproduce, and that's why I haven't addressed it yet).

I do try to keep the UI as slim as possible, but might add screen edge gaps to it - particularly for use with Sonoma widgets.

**Comment by kaiyo655** [comment]
Resolved by v0.7.0
